frontend/src/components/FullWidthAlert.tsx

```diff
@@ -6,7 +6,8 @@ import {
 
 type Props = {
   type: "success" | "warning" | "error" | "info";
-  children: React.ReactNode;
+  heading?: React.ReactNode;
```

Comment:
Adding an optional heading to this element, as the USWDS `Alert` component can be passed one. And with the style changes, I decided to pass the first sentence of the alert as a heading for more attention.